lib/collection/property.js

```diff
@@ -170,6 +170,17 @@ _.extend(Property, /** @lends Property */ {
      * @returns {String}
      */
     replaceSubstitutions: function (str, variables) {
+        var resolvedStr,
+            replaced = true,
+            loopCounter = 0,
+            onReplace = function (match, token) {
```

Comment:
this function is the core and is closed in a closure and as such it is no longer testable.